README.md

```diff
@@ -54,6 +54,7 @@
 - [sample-programs](https://github.com/TheRenegadeCoder/sample-programs)
 - [Syknapse](https://github.com/Syknapse/Contribute-To-This-Project)
 - [useful-dev-tools](https://github.com/lucasnaja/useful-dev-tools)
+- [belejar-git] (https://github.com/petanikode/belajar-git)
```

Comment:
Can you remove the blank space between the name and the link?

Reply:
I already update it, can you check it?
Thank you.